draft-ietf-jsonpath-base.md

```diff
@@ -1178,17 +1178,17 @@ is between:
       compared using `==` to the corresponding element of the second array, or
     * equal objects, that is objects where:
         * for each member of the first object with name `n` and value `v`, there is a member of the second object
-          with name `n` and value `w` where `v` and `w` yield true when comparsed using `==`, and
+          with name `n` and value `w` where `v` and `w` yield true when compared using `==`, and
         * for each member of the second object with name `n` and value `v`, there is a member of the first object
-          with name `n` and value `w` where `v` and `w` yield true when comparsed using `==`.
+          with name `n` and value `w` where `v` and `w` yield true when compared using `==`.
 * a comparison using the operator `!=` yields true if and only if the comparison
 is not between equal values of the same type.
 
 * a comparison using one of the operators `<`, `<=`, `>`, or `>=` yields true if and only if
 the comparison is between values of the same type which are both numbers or both strings and which satisfy the comparison:
 
-    * numbers in the I-JSON {{-i-json}} range of exact values MUST compare using the normal mathematical ordering;
-      one or both numbers outside that range MAY compare using an implementation specific ordering
+    * numbers conforming to I-JSON {{-i-json}} MUST compare using the normal mathematical ordering;
```

Comment:
I'm not sure numbers can "conform" to I-JSON.
Section 2.2 describes when numbers cannot expected to interoperate; the base-10/base-2 problem is not really addressed (but a concept of "expressing" precision is raised that may be hard to define).

```suggestion
    * numbers expected to interoperate as per {{Section 2.2 of -i-json (I-JSON)}} MUST compare using the normal mathematical ordering;
```